services/application.go

```diff
@@ -64,22 +64,27 @@ func (app *ChainlinkApplication) Start() error {
 		app.Stop()
 		app.Exiter(0)
 	}()
+	go app.listenToRunChannel()
 
 	app.jobSubscriberID = app.HeadTracker.Attach(app.JobSubscriber)
 	app.specSubscriberID = app.HeadTracker.Attach(app.specAndRunSubscriber)
-	return multierr.Combine(app.Store.Start(), app.HeadTracker.Start(), app.Scheduler.Start())
+	return multierr.Combine(app.Store.Start(),
+		app.HeadTracker.Start(),
+		app.Scheduler.Start(),
+		app.ResumeSleepingRuns())
 }
 
 // Stop allows the application to exit by halting schedules, closing
 // logs, and closing the DB connection.
 func (app *ChainlinkApplication) Stop() error {
 	defer logger.Sync()
 	logger.Info("Gracefully exiting...")
+	app.JobSubscriber.Stop()
```

Comment:
Interesting that `JobSubscriber` has a `Stop()` but no `Start`. Seems that's being performed by `app.ResumeSleepingRuns()` and `app.listenToRunChannel()`.